### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by HomelessDinosaur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +45,65 @@ We also know abstraction should mean building on existing layers, not hiding the
...
 > These are supported out of the box, but you can also build [custom providers](https://nitric.io/docs/reference/providers/custom/building-custom-provider) as well
 
-## Example
+---
+
+⭐ Give us a star to help support our work
+
+💿 Install Nitric:
+
+macOS:
```
Comment (davemooreuws) [inline README.md:54]:
```suggestion
**macOS:**
```

Comment (davemooreuws) [reply README.md:54]: Maybe make these bold

File: README.md [hunk 2]
```diff
@@ -14,8 +14,6 @@
   <a href="https://discord.gg/Webemece5C"><img alt="Discord" src="https://img.shields.io/discord/955259353043173427?label=discord&style=for-the-badge"></a>
 </p>
 
-## About Nitric
-
 [Nitric](https://nitric.io) is a multi-language framework, with concise inline infrastructure from code. Modern applications should be robust, productive and a joy to build. Nitric solves common problems building for modern platforms:
 
 - [Easy infrastructure](https://nitric.io/docs/concepts/introduction#infrastructure-from-code-if-c) from code
```
Comment (davemooreuws) [inline README.md:19]: This link and the others should be updated, all old now.

File: README.md [hunk 3]
```diff
@@ -49,12 +45,65 @@ We also know abstraction should mean building on existing layers, not hiding the
 
 > These are supported out of the box, but you can also build [custom providers](https://nitric.io/docs/reference/providers/custom/building-custom-provider) as well
 
-## Example
+---
+
+⭐ Give us a star to help support our work
```
Comment (davemooreuws) [inline README.md:50]:
```suggestion
⭐ **Give us a star** to help support our work
```

File: README.md [hunk 4]
```diff
@@ -49,12 +45,65 @@ We also know abstraction should mean building on existing layers, not hiding the
...
+🕹 See our example apps: [Example Apps Repo](https://github.com/nitrictech/examples).
 
-Creating an API, a bucket with access permissions and writing files to that bucket via a serverless function.
+👋 Any questions? Join our developer community on [Discord](https://nitric.io/chat).
+
+## 🍿 Visual Learner?
+
+To get up to speed quickly, take a look at our [quick intro to Nitric](https://www.youtube.com/watch?v=Hljs7Ei9SIs).
+
+<a href="https://www.youtube.com/watch?v=Hljs7Ei9SIs"><img width="600px" src="https://img.youtube.com/vi/Hljs7Ei9SIs/maxresdefault.jpg
```
Comment (davemooreuws) [inline README.md:87]: This is broken

File: README.md [hunk 5]
```diff
@@ -68,19 +117,31 @@ main.post("/notes/:title", async (ctx) => {
...
+## Want more?
+
+Nitric has full documentation at [nitric.io/docs](https://nitric.io/docs), including concepts, reference documentation for various languages and many tutorials/guides.
 
 - Ask questions in [GitHub discussions](https://github.com/nitrictech/nitric/discussions)
 
 - Join us on [Discord](https://nitric.io/chat)
 
-- Find us on [Twitter](https://twitter.com/nitric_io)
+- Find us on [X](https://twitter.com/nitric_io)
```
Comment (davemooreuws) [inline README.md:142]:
```suggestion
- Find us on [X](https://x.com/nitric_io)
```

File: README.md [hunk 6]
```diff
@@ -14,8 +14,6 @@
   <a href="https://discord.gg/Webemece5C"><img alt="Discord" src="https://img.shields.io/discord/955259353043173427?label=discord&style=for-the-badge"></a>
```
Comment (davemooreuws) [inline README.md:14]:
```suggestion
  <a href="https://nitric.io/chat"><img alt="Discord" src="https://img.shields.io/discord/955259353043173427?label=discord&style=for-the-badge"></a>
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -49,12 +45,66 @@ We also know abstraction should mean building on existing layers, not hiding the
 
 > These are supported out of the box, but you can also build [custom providers](https://nitric.io/docs/reference/providers/custom/building-custom-provider) as well
 
-## Example
+---
+
+⭐ **Give us a star** to help support our work
+
+💿 Install Nitric:
```
Comment (davemooreuws) [inline README.md:52]:
```suggestion
💿 **Install Nitric:**
```

File: README.md [hunk 8]
```diff
@@ -49,12 +45,66 @@ We also know abstraction should mean building on existing layers, not hiding the
 
 > These are supported out of the box, but you can also build [custom providers](https://nitric.io/docs/reference/providers/custom/building-custom-provider) as well
 
-## Example
+---
```
Comment (davemooreuws) [inline README.md:48]:
```suggestion
## Get Started
```

**Review by jyecusch — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -49,12 +45,66 @@ We also know abstraction should mean building on existing layers, not hiding the
...
+```
+
+**Windows**:
+
+```
+scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
+scoop install nitric
+```
+
+🚀 Start building your first app:
```
Comment (davemooreuws) [inline README.md:73]:
```suggestion
🚀 **Start building your first app:**
```

File: README.md [hunk 10]
```diff
@@ -49,12 +45,66 @@ We also know abstraction should mean building on existing layers, not hiding the
...
+
+```
+nitric new
+```
+
+🕹 See our example apps: [Example Apps Repo](https://github.com/nitrictech/examples).
```
Comment (davemooreuws) [inline README.md:79]:
```suggestion
🕹 **See our example apps:** [Example Apps Repo](https://github.com/nitrictech/examples).
```

File: README.md [hunk 11]
```diff
@@ -49,12 +45,66 @@ We also know abstraction should mean building on existing layers, not hiding the
...
 
-Creating an API, a bucket with access permissions and writing files to that bucket via a serverless function.
+👋 Any questions? Join our developer community on [Discord](https://nitric.io/chat).
```
Comment (davemooreuws) [inline README.md:81]:
```suggestion
👋 **Any questions?** Join our developer community on [Discord](https://nitric.io/chat).
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: README.md [hunk 12]
```diff
@@ -68,19 +118,31 @@ main.post("/notes/:title", async (ctx) => {
...
+4. **Automated Infrastructure, Best Practices Included** One of the most error-prone aspects of cloud development is managing permissions, configurations, and security policies. Nitric automates this, making security best practices—like least privilege access and proper service configurations easy.
 
-## Get in touch
+5. **Focus on Application Logic** Nitric's approach allows you to focus on building your application, instead of the scaffolding required to run it in the cloud. By removing the manual steps from the IaC process, Nitric eliminates significant boilerplate and reduces the runtime checking needed to handle configuration errors.
+
+6. **Plugin-Based Architecture** Nitric's plugin-based architecture allows you to use the deployment plugins we provide, which use Pulumi or Terraform for deployment, or write your own. This flexibility allows you to use the tools you're comfortable with, while still benefiting from Nitric's infrastructure automation and cloud-agnostic approach.
+
+## Want more?
+
+Nitric has full documentation at [nitric.io/docs](https://nitric.io/docs), including concepts, reference documentation for various languages and many tutorials/guides.
```
Comment (davemooreuws) [inline README.md:137]:
```suggestion
Nitric has full documentation at [nitric.io/docs](https://nitric.io/docs), including concepts, reference documentation for various languages and many [tutorials/guides](https://nitric.io/docs/guides).
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

File: README.md [hunk 13]
```diff
@@ -49,12 +45,68 @@ We also know abstraction should mean building on existing layers, not hiding the
 
 > These are supported out of the box, but you can also build [custom providers](https://nitric.io/docs/reference/providers/custom/building-custom-provider) as well
 
-## Example
+## 🧑‍💻 Get started
+
+💿 Install Nitric:
```
Comment (davemooreuws) [inline README.md:50]:
```suggestion
💿 **Install Nitric:**
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**